src/conv.rs

```diff
@@ -86,7 +98,7 @@ impl Conv for Severity {
     }
 }
 
-impl ConvWith<&LineIndex> for &AtomTextEdit {
+impl ConvWith<&LineIndex> for &Indel {
```

Comment:
Urgh, we probably should replace it with to_proto/from_proto pattren, like we did in rust-analyzer proper.

Reply:
Fixed.